Pod/Classes/LSTracer.h

```diff
@@ -59,6 +59,9 @@ extern NSInteger const LSBackgroundTaskError;
 /// `LSTracer` instance's globally unique id ("guid"), and assigned automatically by LightStep.
 @property(nonatomic, readonly) UInt64 runtimeGuid;
 
+/// HTTP session to be used for performing requests. This enable sharing a connnection pool with your own app.
```

Comment:
I think that would be great, but I know you are busy, etc. We can file an issue if you'd rather.